### ISSUE TITLE
file.recurse - Unable to manage file: 'charmap' codec can't decode byte 0x81 in position 6107: character maps to <undefined>

### ISSUE THREAD

**Issue by otm1** [saltstack/salt#44960]
When using file.recurse I get the following error on some of the files:
`Unable to manage file: 'charmap' codec can't decode byte 0x81 in position 6107: character maps to <undefined>`
 
I had this error on salt master 2016 and i upgraded to 2017.7.2 in hopes that it would resolve the issue.

My SLS file looks like this: 
```
copy_build:
  file.recurse:
    - name: c:\test1\
    - source: salt://repo/test1
    - include_empty: True
    - clean: True
```

I have tried using different encodings like ascii and latin-1 by using the follwing:
`    - encoding_type: ascii`
but it doesn't seem to do anything.

The salt minion is a Windows 2012 R2 server with 2017.7.2 and Python 3.

Salt master:
```
Salt Version:                                             
           Salt: 2017.7.2                                 
                                                          
Dependency Versions:                                      
           cffi: Not Installed                            
       cherrypy: 3.5.0                                    
       dateutil: 2.5.3                                    
      docker-py: Not Installed                            
          gitdb: 2.0.0                                    
      gitpython: 2.1.1                                    
          ioflo: Not Installed                            
         Jinja2: 2.8                                      
        libgit2: Not Installed                            
        libnacl: Not Installed                            
       M2Crypto: Not Installed                            
           Mako: 1.0.6                                    
   msgpack-pure: Not Installed                            
 msgpack-python: 0.4.8                                    
   mysql-python: Not Installed                            
      pycparser: Not Installed                            
       pycrypto: 2.6.1                                    
   pycryptodome: Not Installed                            
         pygit2: Not Installed                            
         Python: 2.7.13 (default, Jan 19 2017, 14:48:08)  
   python-gnupg: Not Installed                            
         PyYAML: 3.12                                     
          PyZMQ: 16.0.2                                   
           RAET: Not Installed                            
          smmap: 2.0.1                                    
        timelib: Not Installed                            
        Tornado: 4.4.2                                    
            ZMQ: 4.2.1                                    
                                                          
System Versions:                                          
           dist: debian 9.0                               
         locale: UTF-8                                    
        machine: x86_64                                   
        release: 4.9.0-2-amd64                            
         system: Linux                                    
        version: debian 9.0                               
```

**Comment by Ch3LL** [comment]
Do you have any special characters in those files? Do you know which file it is that is causing this? I was hoping to try to replicate this but need to know what the files look like that are causing this issue. Thanks

**Comment by otm1** [comment]
Thank you. I have attached two files. One file is already in the destination folder. The other file is the new file which should replace the old file.

[FileAlreadyInDestinationFolder.zip](https://github.com/saltstack/salt/files/1562488/FileAlreadyInDestinationFolder.zip)
[FileInRepository.zip](https://github.com/saltstack/salt/files/1562489/FileInRepository.zip)

There are many other files failing but this is for certain one of them.

**Comment by otm1** [comment]
Hi, How can I unblock this issue (the information requested has been provided)?

**Comment by otm1** [comment]
Did not know how to unblock the issue. Created new issue #45873
